src/Barryvdh/TranslationManager/Manager.php

```diff
@@ -40,62 +41,96 @@ public function missingKey($namespace, $group, $key)
     public function importTranslations($replace = false)
     {
         $counter = 0;
-        foreach($this->files->directories($this->app->make('path').'/lang') as $langPath){
+        foreach ($this->files->directories($this->app->make('path') . '/lang') as $langPath) {
             $locale = basename($langPath);
+            $counter += $this->importDirectory($langPath, $replace, $locale, $counter);
```

Comment:
should it be ` $counter += $this->importDirectory($langPath, $replace, $locale);` ?